package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "react-nimble-survey",
+  "version": "0.1.0",
+  "private": true,
+  "engines" : {
+    "npm" : ">=8.11.0",
+    "node" : ">=16.16.0"
+  },
+  "dependencies": {
```

Comment:
@rosle Should we use `save-exact=true` to prevent `^` after adding new dependencies?